media/logo.svg

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+
+<svg
```

Comment:
I have an SVG optimizer that might be able to shrink this guy by quite a bit, but I'll investigate that separately.